api.go

```diff
@@ -327,6 +327,59 @@ func extractMessage(data []byte) (*Message, error) {
 	return resp.Result, nil
 }
 
+func messageIDs(msgs []Editable) (string, int64) {
```

Comment:
- `func embedMessages(params map[string]string, msgs []Editable)`
- Move to the `options.go`